### PR TITLE
add variables for new invoice template

### PR DESCRIPTION
**Pull request by nickpell**
For http://manage.dimagi.com/default.asp?157984

@benrudolph

### CODE REVIEW

**Comment by benrudolph** [comment]
looks good. just checking, the actual variables have been added to the server?

**Comment by benrudolph** [comment]
(merging cause the comment doesn't make a different)

**Comment by nickpell** [comment]
@benrudolph Not yet, going to try that now that this is merged (first time working on ansible)

File: ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 1]
```diff
@@ -489,3 +489,6 @@ SMS_PREFIXES_BY_COUNTRY = '{{ localsettings.SMS_PREFIXES_BY_COUNTRY }}'
 # See corehq/apps/builds/README.md for more information.
 BASE_ADDRESS = '{{ SITE_HOST }}'
 EMAIL_USE_TLS = {{ localsettings.EMAIL_USE_TLS }}
+
+BANK_ROUTING_NUMBER_ACH = '{{ localsettings.BANK_ROUTING_NUMBER_ACH }}'
+BANK_ROUTING_NUMBER_WIRE = '{{ localsettings.BANK_ROUTING_NUMBER_WIRE }}'
```
Comment (czue) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: fyi @nickpell @benrudolph  if you add an environment variable like this you need to make sure it exists in all environments (this was missing from india) or ansible will choke on it. you can do a more protective style like #70 if you don't want to add it everywhere

Comment (nickpell) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: ahh, thanks

Comment (czue) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: if we need to add something specific we should do that - i just deployed leaving them blank

Comment (nickpell) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: that's fine, most of billing (invoicing in particular) is unused on india.

Comment (benrudolph) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: i was wondering what all those if statements were supposed to be doing

**Pull request merged**